src/mpyl/schema/run_properties.schema.yml

```diff
@@ -108,3 +108,6 @@ properties:
           type: string
         icon:
           type: string
+        parallelism_factor:
```

Comment:
It's a nit, but the word `parallelism` would be enough. In the description you could explain further.